convert2rhel/pkghandler.py

```diff
@@ -662,6 +663,30 @@ def remove_non_rhel_kernels():
     return non_rhel_kernels
 
 
+def fix_default_kernel():
+    """
+    Systems converted from Oracle Linux or CentOS Plus may have leftover kernel-uek or kernel-plus in
+    /etc/sysconfig/kernel as DEFAULTKERNEL.
+    This function fixes that by replacing the DEFAULTKERNEL setting from kernel-uek or kernel-plus to rhel-kernel
+    """
+    loggerinst = logging.getLogger(__name__)
+
+    loggerinst.info("Checking for incorrect kernel assignment...")
+    kernel_sys_cfg = utils.get_file_content("/etc/sysconfig/kernel")
+
+    kernel_to_change = ["kernel-uek", "kernel-plus"]
+
+    if any(kernel in kernel_sys_cfg for kernel in kernel_to_change):
```

Comment:
@ZhukovGreen better way to do this?

```suggestion
    possible_kernels = ["kernel-uek", "kernel-plus"]
	kernel_to_change = next(iter(kernel for kernel in possible_kernels if kernel in kernel_sys_cfg), None)
    if kernel_to_change:
```